src/main/components/Browse/Dashboard.js

```diff
@@ -25,82 +25,69 @@ const CustomTableCell = withStyles(theme => ({
   }
 }))(TableCell);
 
-class Dashboard extends Component {
-  static propTypes = {
-    analyses: PropTypes.array.isRequired,
-    classes: PropTypes.object.isRequired,
-    columnNames: PropTypes.array.isRequired,
-    selectAnalysis: PropTypes.func.isRequired,
-    selectedAnalysis: PropTypes.string,
-    title: PropTypes.string.isRequired
-  };
-
-  constructor(props) {
-    super(props);
-    this.handleAnalysisClick = this.handleAnalysisClick.bind(this);
-  }
-
-  handleAnalysisClick = () => this.props.onAnalysisClick();
-
-  render() {
-    const {
-      analyses,
-      classes,
-      columnNames,
-      selectAnalysis,
-      selectedAnalysis,
-      title
-    } = this.props;
-
-    const analysisItems = analyses.map(analysis => {
-      const isSelected = selectedAnalysis === analysis.id;
-      const onAnalysisClick = () => {
-        if (!isSelected) {
-          selectAnalysis(analysis.id, title);
-        }
-        this.handleAnalysisClick();
-      };
-      const formatIdList = idList => idList.join("\n");
-      analysis = {
-        ...analysis,
-        libraryIds: formatIdList(analysis.libraryIds),
-        sampleIds: formatIdList(analysis.sampleIds)
-      };
-      return (
-        <AnalysisItem
-          analysis={analysis}
-          isSelected={isSelected}
-          key={analysis.title}
-          onClick={onAnalysisClick}
-        />
-      );
-    });
-
+const Dashboard = ({
+  analyses,
+  classes,
+  columnNames,
+  handleAnalysisClick,
+  selectAnalysis,
+  selectedAnalysis,
+  title
+}) => {
+  const analysisItems = analyses.map(analysis => {
+    const isSelected = selectedAnalysis === analysis.id;
+    const onAnalysisClick = () => {
+      if (!isSelected) {
+        selectAnalysis(analysis.id, title);
+      }
+      handleAnalysisClick();
+    };
+    const formatIdList = idList => idList.join("\n");
+    analysis = {
+      ...analysis,
+      libraryIds: formatIdList(analysis.libraryIds),
```

Comment:
Also be careful here -- remember that we abstracted the column names so that the components downstream of Browse wouldn't have to remember specific column names. Here, you're hardcoding some back in.